### DISCUSSION TITLE
regrading rmcell

### DISCUSSION THREAD

**Discussion by anujyadav0693** [neperfepx/neper#878] [Q&A] [answered]
respected sir,
I am using the rmcell command in version 4.9.1 still it is not working.

anuj@DESKTOP-41K2B3H:/home/model$ neper -T -n from_morpho -morpho "diameq:lognormal(0.1,0.01)+0.5*lognormal(0.01,0.001),1-sphericity:lognormal(0.145,0.03)" -transform "rmcell(diameq<=0.01)" -dim 2 -domain "square(1,2)"

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.9.1-2
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 16 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/anuj/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -T loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] -n from_morpho -morpho
         diameq:lognormal(0.1,0.01)+0.5*lognormal(0.01,0.001),1-sphericity:lognormal(0.145,0.03)
         -transform rmcell(diameq<=0.01) -dim 2 -domain square(1,2)
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   : Creating domain...
Info   : Creating tessellation...
Info   :       . Number of cells: 376
Info   :   - Setting seeds... 100%
Info   :   - Running tessellation...
Info   :     > Initial solution: f   =1.270452870
Info   :     > Iteration 284447: fmin=0.000299566 f=0.000299567
Info   :     > Final solution  : f   =0.000299566 (284447 iterations)
Info   :     > Reached `eps' criterion.
Info   : Generating crystal orientations...
Info   : Transforming tessellation...
Info   :   - Removing cells...
Aborted (core dumped)

code is aborted. But i am using this command   neper -T -n 2 -transform "rmcell(id>1)"
 Then it is working. What problem is it? Please tell me. This bug is still not properly removed.

anuj@DESKTOP-41K2B3H:~/Progs/neper-4.9.0/src/build/models$ neper -T -n 2 -transform "rmcell(id>1)"

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.9.1-2
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt|libscotch (full)
Info   : Running on 16 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2022, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/anuj/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -T loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] -n 2 -transform rmcell(id>1)
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   : Creating domain...
Info   : Creating tessellation...
Info   :   - Setting seeds...
Info   :   - Running tessellation...
Info   : Generating crystal orientations...
Info   : Transforming tessellation...
Info   :   - Removing cells...
Info   :     > Removed 1 cells...
Info   : Writing results...
Info   :     [o] Writing file `n2-id1.tess'...
Info   :     [o] Wrote file `n2-id1.tess'.
Info   : Elapsed time: 0.048 secs.

thank you

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
